packages/button/src/Button.tsx

```diff
@@ -1,50 +1,78 @@
 import React from 'react';
 import cn from 'classnames';
 
-import type { ButtonProps } from './Button.types';
+import ButtonIcon from './components/ButtonIcon';
+import ButtonLoader from './components/ButtonLoader';
+import { ButtonProps } from './Button.types';
 
 import styles from './styles/default.module.css';
 import stylesSize from './styles/size.module.css';
 import stylesVariant from './styles/variant.module.css';
 
-const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+const Button = React.forwardRef<
```

Comment:
Is it alright to use arrow functions for components? If we're following the [styleguide convention](https://github.com/hey-car/hey-ui/blob/develop/STYLEGUIDE_CONVENTION.md#jsx), we're using named functions